### PR TITLE
Rf 9-10 planner modal

### PR DESCRIPTION
**Pull request by Anelia94**

### CODE REVIEW

File: client/src/app/menu-planner/planner-modal/planner-modal.js [hunk 1]
```diff
@@ -1,95 +1,78 @@
...
-  }
+
+  useEffect(() => {
+    if (!singleMeal) {
+      return;
+    }
+    const currentDailyMenu = dailyMenu;
+    currentDailyMenu[singleMeal] = selectedRecipe;
+    setDailyMenu({ ...currentDailyMenu });
+  }, [selectedRecipe]);
```
Comment (bancho22) [inline client/src/app/menu-planner/planner-modal/planner-modal.js:34]: New line after 34

File: client/src/app/shared-components/recipe-cards/recipe-Cards-style.scss [hunk 2]
```diff
@@ -48,29 +48,29 @@
   }
   
 }
-.modal-dialog {
-  max-width: 60rem;
-  .modal-content {
-    width: 60rem;
+// .modal-dialog {
```
Comment (bancho22) [inline client/src/app/shared-components/recipe-cards/recipe-Cards-style.scss:51]: Let's delete this file, if we don't need it anymore (&& will not need it in the future)

File: client/src/app/menu-planner/planner-modal/planner-modal.js (hunk 1 above)
Comment (RumyanaA) [reply client/src/app/menu-planner/planner-modal/planner-modal.js:34]: resolved

File: client/src/app/shared-components/recipe-cards/recipe-Cards-style.scss (hunk 2 above)
Comment (RumyanaA) [reply client/src/app/shared-components/recipe-cards/recipe-Cards-style.scss]: deleted comments

File: client/src/app/menu-planner/menu-planner.js [hunk 3]
```diff
@@ -13,43 +15,26 @@ function MenuPlanner() {
   const [isOpen, setIsOpen] = useState(false);
   const [dateToShow, setDateToShow] = useState(new Date());
   const [date, setDate] = useState(new Date());
+  const [currMenu, setCurrMenu] = useState({});
```
[the remark this answers is not in the document]
Comment (RumyanaA) [reply client/src/app/menu-planner/menu-planner.js]: resolved

**Review by RumyanaA — APPROVED** [review]

**Pull request merged**